cosmic_ray/operators/remove_decorator.py

```diff
@@ -0,0 +1,24 @@
+import ast
+
+from .operator import Operator
+
+
+class RemoveDecorator(Operator):
+    """An operator that removes each of the non standard decorators."""
+    regular_decorators = frozenset(["classmethod", "staticmethod", "abstractmethod"])
+
+    def visit_FunctionDef(self, node):  # noqa
+        decorator_candidates = [x for x in node.decorator_list if x.id not in self.regular_decorators]
+        if decorator_candidates:
+            return self.visit_mutation_site(node, len(decorator_candidates))
+
+        return node
+
+    def mutate(self, node, idx):
+        """Modify the decorator list to remove one decorator at each mutation"""
+        decorator_candidates = [x for x in node.decorator_list if x.id not in self.regular_decorators]
+        del decorator_candidates[idx]
+        modified_decorator = decorator_candidates + [x for x in node.decorator_list if x.id in self.regular_decorators]
```

Comment:
I think something like this will work:
```python
def mutate(self, node, idx):
    """Modify the decorator list to remove one decorator at each mutation"""
    candidates = [(i, d)
                  for (i, d) in enumerate(node.decorator_list)
                  if d.id not in self.regular_decorators]
    remove_at, _ = candidates[idx]
    del node.decorator_list[remove_at]
    return node
```

**This is completely untested**,  so please verify it does (or doesn't) do the right thing.